src/yuzu/main.cpp

```diff
@@ -1035,19 +1035,36 @@ void GMainWindow::BootGame(const QString& filename) {
     }
 
     const u64 title_id = Core::System::GetInstance().CurrentProcess()->GetTitleID();
+    const FileSys::PatchManager patch_manager(title_id);
+    QString dlc;
+
+    for (const auto& kv : patch_manager.GetPatchVersionNames(nullptr)) {
+        if (kv.first == "DLC" && !kv.second.empty()) {
+            dlc = QString::fromStdString("DLC " + kv.second);
+        }
+    }
```

Comment:
Volca brought this up, there can be multiple DLC installed so this was suggested:
```suggestion
    u32 num_dlc{};

    for (const auto& [type, version] : patch_manager.GetPatchVersionNames(nullptr)) {
        if (type == "DLC" && !version.empty()) {
            ++num_dlc;
        }
    }
```

Reply:
```cpp
const QString dlc = (num_dlc == 0) ? tr("No DLC") : ("DLC: " + num_dlc);
```

Reply:
```cpp
const auto patch_names = patch_manager.GetPatchVersionNames(nullptr);
const auto num_dlc = std::count_if(patch_names.cbegin(), patch_names.cend(),
                                   [](const auto& entry) { return entry.first == "DLC" && !entry.second.empty(); });
```